medusa/tv/series.py

```diff
@@ -1048,8 +1048,8 @@ def make_ep_from_file(self, filepath):
                     if not cur_ep:
                         raise EpisodeNotFoundException
                 except EpisodeNotFoundException:
-                    logger.error(u'{indexerid}: Unable to figure out what this file is, skipping {filepath}',
-                                 indexerid=self.indexerid, filepath=filepath)
+                    logger.warning(u'{indexerid}: Episode not found for this show. Please full update show. Skipping {filepath}',
```

Comment:
What was missed was imprecise wording: `Please run a full update.` 
This is ambiguous and could be taken to mean a full library update instead of the intended series update.  On a large library that could mean the difference between a few minutes to update to several hours (or even more) to update.